test/unit_tests/braket/experimental/algorithms/qc_qmc/test_qc_qmc.py

```diff
@@ -0,0 +1,159 @@
+from unittest.mock import patch
+
+import pennylane as qml
+import pytest
+from pennylane import numpy as np
+
+from braket.experimental.algorithms.qc_qmc.classical_qmc import (
+    chemistry_preparation,
+    classical_qmc,
+    full_imag_time_evolution_wrapper,
+    hartree_fock_energy,
+)
+from braket.experimental.algorithms.qc_qmc.qc_qmc import q_full_imag_time_evolution_wrapper, qc_qmc
+
+np.set_printoptions(precision=4, edgeitems=10, linewidth=150, suppress=True)
+
+
+@pytest.fixture
+def qmc_data():
+    symbols = ["H", "H"]
+    geometry = np.array([[0.0, 0.0, 0.0], [0.0, 0.0, 1.41729459]], requires_grad=False)
+    mol = qml.qchem.Molecule(symbols, geometry, basis_name="sto-3g")
+    trial = np.array([[1, 0], [0, 1], [0, 0], [0, 0]])
+    prop = chemistry_preparation(mol, geometry, trial)
+    dev = qml.device("lightning.qubit", wires=4)
+    e_hf = hartree_fock_energy(trial, prop)
+    return (trial, prop, dev, e_hf)
+
+
+def trial_state_circuit() -> None:
```

Comment:
nit: inconsistent about -> None in this file. Not convinced it's necessary for tests

Reply:
Actually nvm, I see this isn't a test function. Seems good to me then